k8s/hpas/apache-tika-hpa.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
```

Comment:
don't know much about this yet, but I feel like everything we're going to `apply_infra` it's going to revert to 1 replica due to it being part of the deployment.
Might want to research quickly about it if not done already 👍

Reply:
Yes it does. I'll refine once I've better sense of how many pods are usually needed, and look into patching over applying.

Reply:
Ok I've updated the `apply_infra` script to preserve the current number of replicas.

Reply:
niceeee